docs/deepsemgrep/semgrep-pro-data-flow.md

```diff
@@ -0,0 +1,23 @@
+---
+slug: semgrep-pro-data-flow
+append_help_link: true
+description: "This article documents data flow analysis implementation in Semgrep Cloud Platform and helps you to enable and find it on the Findings page."
+hide_title: false
+title: Semgrep Pro Engine taint traces
+---
+
+import MoreHelp from "/src/components/MoreHelp"
+import DisplayTaintedDataIntro from "/src/components/concept/_cloud-platform-display-tainted-data.mdx"
+import DisplayTaintedDataProcedure from "/src/components/procedure/_cloud-platform-display-tainted-data.mdx"
+
+## Introduction
+
+This article documents the data-flow analysis implementation in the Semgrep Cloud Platform and helps you to enable it, and provides a brief overview of the benefits of interfile analysis capabilities of the Semgrep Pro Engine compared to the analysis of the Semgrep Code.
```

Comment:
Update: This article documents Semgrep Pro Engine's dataflow analysis and interfile analysis in the Cloud Platform. It helps you to enable it and provides a overview of the benefits compared to the analysis of Semgrep OSS.